Library/ViewModels/LoginToutViewModel.swift

```diff
@@ -220,9 +269,24 @@ public final class LoginToutViewModel: LoginToutViewModelType, LoginToutViewMode
   public var inputs: LoginToutViewModelInputs { return self }
   public var outputs: LoginToutViewModelOutputs { return self }
 
-  fileprivate var viewWillAppearProperty = MutableProperty(())
-  public func viewWillAppear() {
-    self.viewWillAppearProperty.value = ()
+  fileprivate let appleLoginButtonPressedProperty = MutableProperty(())
+  public func appleLoginButtonPressed() {
+    self.appleLoginButtonPressedProperty.value = ()
+  }
+
+  // This property is being set as lazy because we can't use @available in computed properties.
+  @available(iOS 13.0, *)
```

Comment:
Do you think it's necessary to wrap these outputs with the `@available` check? I would imagine that as long as you can make sure the inputs are not being triggered on the VC, that it would be safe not to do `@available` checks in the VM, since you're not actually using any unavailable APIs here 🤔. 

Thoughts @justinswart ?

Reply:
You're right. But I think that I added the availability because previously I was testing the whole flow (fetching v1 user, etc) and the compiler was complaining, I think due the involvement of other outputs and signals. But, because this is not happening on this PR, I will remove it.